docs/source/diagrams/ReflectometryReductionOne_LowLvl-v1_wkflw.dot

```diff
@@ -32,17 +41,19 @@ $decision_style
   checkCorrAlg    [label="CorrectionAlgorithm?"]
   checkSourceRotation [label="Is Source in Correct Position?"]
   checkSourceRotationBack [label = "Was Source Rotated?"]
-  checkRebinMin  [label="MomentumTransferMinimum given?"]
-  checkRebinMax [label="MomentumTransferMaximum given?"]
+  checkRebinMin   [label="MomentumTransferMinimum given?"]
+  checkRebinMax   [label="MomentumTransferMaximum given?"]
   checkRebinStep  [label="MomentumTransferStep given?"]
   checkScale      [label="ScaleFactor given?"]
+  checkMonUnity   [label="Monitors?"]
+  checkBgMinMax   [label="Min or Max\nnon-zero?"]
+  monIntWavInterval [label="Monitor Integration\nWavelength Interval"]
```

Comment:
I think this should be split into two different input properties (also, note that they should be defined as `param` instead of `decisions`), and called `MonitorIntegrationWavelengthMin` and `MonitorIntegrationWavelengthMax`, which are the names of the input properties we are using.